torch/onnx/utils.py

```diff
@@ -1654,7 +1654,7 @@ def _export_file(
             for k, v in export_map.items():
                 z.writestr(k, v)
     elif export_type == _exporter_states.ExportTypes.DIRECTORY:
-        if isinstance(io.BytesIO, f) or not os.path.isdir(f):  # type: ignore[arg-type]
+        if isinstance(io.BytesIO, type(f)) or not os.path.isdir(f):  # type: ignore[arg-type]
```

Comment:
Is this right?

Reply:
I had error message saying f doesn't fit the input requirement of isinstance()

Reply:
Done